### DISCUSSION TITLE
Disentangling failed

### DISCUSSION THREAD

**Discussion by jaktykusuma** [Kinggerm/GetOrganelle#144] [Q&A]
Hello,

I'm using my own dataset and it generated multiple warnings on disentangling. What could be the problem? I'm kind of new of this analysis, so any inputs will be really helpful.

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8125055/get_org.log.txt)
![K115](https://user-images.githubusercontent.com/63284031/155339868-d090de16-7148-4770-848d-b86b90de5617.png)

**Comment by Kinggerm** [comment]
According to the log file, you have relatively low coverage for the plastome, which may be why the graph is broken (organelle insufficient).

Also, please turn on the depth and length in Bandage.
